docs/language-server/protocol-language-server.md

```diff
@@ -376,6 +376,11 @@ interface ExpressionUpdate {
    *  - array with multiple values represents an intersetion type
    */
   type: string[];
+  /**
+   * The list of types this expression can be converted to.
+   */
+  hiddenType: string[];
```

Comment:
Shouldn't we use plural? E.g. make it`hiddenTypes`?

Reply:
From the user's perspective, it's either a type or an intersection type. That's why I went with a singular name